StoichWeb.py

```diff
@@ -0,0 +1,154 @@
+__author__ = 'Zhenbang Yu'
+__copyright__ = 'Copyright 2022, Zhenbang Yu'
+__email__ = 'roger_yu@berkeley.edu'
+
+
+# TODO: Implement Save Input
+# TODO: Implement Save Results
+# TODO: Implement Save All
+# TODO: Implement Open Input
+
+# import flask relevant packages.
+from flask import Flask, url_for, render_template, redirect, request
+
+# import useful packages
+import json
+import pickle
+import numpy as np
+import os
+from collections import OrderedDict
+import pandas as pd
+from matplotlib.figure import Figure
+import mpld3
+
+# import useful packages in original StoichiometryFitter
+import ReportResults
+import imp
+import PhysicsBasics as pb
+import CountsToQuant
+import PhaseFit
+
+from ternary_diagram import TernaryDiagram
+
+# Stoich = pd.read_csv("ConfigData/stoich Silicates.csv")
+Stoich = np.genfromtxt('ConfigData/stoich Silicates.csv', dtype=None, comments='#', delimiter=',', skip_header=1, converters={1: lambda s: float(s)})
+
+# initiate flask app
+app = Flask(__name__)
+
+# Everything in here.
+@app.route('/', methods = ["POST","GET"])
+def login():
+    if request.method == "POST":
+        # Detect not identified POST requests.
+        if 'GO' not in request.form:
+            return "<h1>There is a unidentified POST request!!</h1>"
+
+        # Detect if the user click the submit button.
+        if request.form['GO'] == "Go!":
+            # Use to store matplotlib figures in html string, initiate as empty string.
+            Fig1 = ""
+            Fig2 = ""
+
+            Counts = np.zeros(118)
+            # Obtain InputType
+            inputType = request.form["IType"]
+
+            # Read in ElementsList
+            element = ["H","He","Li","Be","B","C","N","O","F","Ne","Na","Mg","Al","Si","P","S","Cl","Ar","K","Ca","Sc","Ti","V","Cr","Mn","Fe","Co","Ni","Cu","Zn","Ga","Ge","As","Se","Br","Kr","Rb","Sr","Y","Zr","Nb","Mo","Tc","Ru","Rh","Pd","Ag","Cd","In","Sn","Sb","Te","I","Xe","Cs","Ba","La","Ce","Pr","Nd","Pm","Sm","Eu","Gd","Tb","Dy","Ho","Er","Tm","Yb","Lu","Hf","Ta","W","Re","Os","Ir","Pt","Au","Hg","Tl","Pb","Bi","Po","At","Rn","Fr","Ra","Ac","Th","Pa","U","Np","Pu","Am","Cm","Bk","Cf","Es","Fm","Md","No","Lr","Rf","Db","Sg","Bh","Hs","Mt","Ds","Rg","Cn","Uut","Fl","Uup","Lv","Uus","Uuo"]
```

Comment:
Just FYI, this is already in physics basics.  By duplicating the code you open the possibility to hard to debug bugs later.  for example, if there were a typo in any one of the elements, then everything would work fine, but that element would give incorrect quantifications until someone noticed -- which could take years and require sending corrections to papers.  When possible, always reuse existing tested code rather than write new code even when it seems trivial.